puzzles/migrations/0024_auto_20210104_2026.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.1.4 on 2021-01-04 20:26
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('puzzles', '0023_auto_20201221_0030'),
```

Comment:
Should run black on this -- otherwise presubmit will complain about the single quotes.